community/modules/scheduler/htcondor-access-point/versions.tf

```diff
@@ -29,5 +29,5 @@ terraform {
     module_name = "blueprints/terraform/hpc-toolkit:htcondor-access-point/v1.20.0"
   }
 
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.1"
```

Comment:
Just curious what this upgrade was for - it seems like a significant jump.

Reply:
The use of `nullable = false` introduces a requirement for Terraform 1.1 or above.